plugins/polio/tests/test_vaccine_stock_archive_task.py

```
@@ -0,0 +1,706 @@
+from beanstalk_worker.services import TestTaskService
```

Ruff (0.8.0)

`beanstalk_worker.services.TestTaskService` imported but unused

Remove unused import: `beanstalk_worker.services.TestTaskService`

(F401)